setup.cfg

```
@@ -1,59 +1,103 @@
 [metadata]
 name = ndcube
+provides = ndcube
 author = The SunPy Community
 author_email = sunpy@googlegroups.com
 license = BSD 2-Clause
 license_file = LICENSE.rst
-url = http://docs.sunpy.org/projects/ndcube/
+url = https://docs.sunpy.org/projects/ndcube/en/stable/
```

It will redirect to stable by default, but if we were to ever change it then it would redirect there:
```suggestion
url = https://docs.sunpy.org/projects/ndcube/
```